src/lib/integration_test_cloud_engine/coda_automation.ml

```diff
@@ -90,7 +91,7 @@ module Network_config = struct
     in
     assoc
 
-  let expand ~logger ~test_name ~(cli_inputs : Cli_inputs.t)
+  let expand ~logger ~test_name ~(cli_inputs : Cli_inputs.t) ~(debug : bool)
```

Comment:
I just took the "path of least resistance" but I'm sure there's a cleaner way where we roll-in the debug flag with cli_inputs, although it may take somewhat of a refactor

Reply:
Nah, this works for now. If anything, we could have `Engine_cli_inputs` vs `Executive_cli_inputs`, but we wouldn't want to put debug into this `Cli_inputs` because it is opaque and specific to the cloud engine.

Reply:
ah that does explain a few things, cool